ignite/contrib/handlers/base_logger.py

```diff
@@ -103,7 +103,26 @@ def _setup_output_metrics(self, engine: Engine) -> Dict[str, Any]:
                 output_dict = {"output": output_dict}
 
             metrics.update({name: value for name, value in output_dict.items()})
-        return metrics
+
+        metrics_dict = OrderedDict()
+
+        for name, value in metrics.items():
+            if value is None:
+                continue
+            if isinstance(value, numbers.Number):
+                metrics_dict[f"{self.tag}/{name}"] = value
+            elif isinstance(value, torch.Tensor) and value.ndimension() == 0:
+                metrics_dict[f"{self.tag}/{name}"] = value.item()
+            elif isinstance(value, torch.Tensor) and value.ndimension() == 1:
+                for i, v in enumerate(value):
+                    metrics_dict[f"{self.tag}/{name}/{i}"] = v.item()
+            else:
+                if isinstance(value, str) and log_text:
```

Comment:
Since only some loggers allow text, I have handled it using a flag like this. Let me know if there is a better way for doing this.

Reply:
I think it is OK, thanks !